### ISSUE TITLE
Investigate Bootstrapped DQNs and their precise implementation

### ISSUE THREAD

**Issue by raharth** [raharth/PyTorch_Lib#38]
There might also be variations to this as applying Dropout, or using partitions of the data to train the different heads. It might also be an option to combine Bootstrapping (i.e. multiple heads on a single network learning the same targets) with ensemble techniques.
